### PR TITLE
Implement events page

### LINKED ISSUE #11: Implement events page
- [x] events use a template "event" - setup via meta data sheet for all docs in the /events/*
- [x] short version of header/hero with only breadcrumb and title
- [x] implement event block for the event details
- [x] event data stored into meta data (region, event type) > use for index
- [ ] support for extra event data, see below > can be via extra issues
  - [x] simple event 
  - [ ] event with related product cards (will be addressed in branch event-products)
  - [ ] event with related resources (will be addressed with #272)
- [ ] implement add to calender button (will be addressed in its own PR)
- [x] add missing footer curve to publication, news and the event page

Docs to test with:
- simple event https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B6C7A7E29-43E1-4360-B560-2A8680402BBD%7D&file=bps-biophysical-society.docx&action=default&mobileredirect=true
- event with related product cards https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7BD05340AE-147F-4BF6-96CD-52E21CD880E7%7D&file=world-pharma-week-2020.docx&action=default&mobileredirect=true
- event with related resources https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B77F8C233-1CBB-4E53-9CC4-D748DFBEECD3%7D&file=pegs-europe-conference.docx&action=default&mobileredirect=true > the related resources block is via a different issue

Samples:
- https://www.moleculardevices.com/events/imaging-user-meeting-2023
- https://www.moleculardevices.com/events/focus-on-microscopy-fom
- https://www.moleculardevices.com/events/mps-world-summit
- https://www.moleculardevices.com/events/cellbio-ascb

### PR DESCRIPTION
**Pull request by lydiapuric**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #11

Test URLs 
Simple Version (without Add Calendar functionality):
- Before: https://main--moleculardevices--hlxsites.hlx.page/events/bps-biophysical-society
- After: https://event--moleculardevices--hlxsites.hlx.page/events/bps-biophysical-society

AND
Version with Products (note: products not part of this PR, will be a separate PR)
- Before: https://main--moleculardevices--hlxsites.hlx.page/events/cell-bio
- After: https://event--moleculardevices--hlxsites.hlx.page/events/cell-bio

### CODE REVIEW

**Comment by jose-correia** [comment]
Did we decide on leaving the Add To Calendar button out for now?

File: blocks/event-summary/event-summary.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { formatDate } from '../../scripts/scripts.js';
+import {
+  div, h3, li, p, ul,
+} from '../../scripts/dom-helpers.js';
+
+export default async function decorate(block) {
+  let startDate = getMetadata('start-date');
+  if (startDate) { startDate = formatDate(startDate); }
+  // eslint-disable-next-line prefer-destructuring
+  if (startDate) { startDate = startDate.split(',')[0]; }
```
Comment (jose-correia) [inline blocks/event-summary/event-summary.js:11]: this if logic can be included in the previous one

**Review by jose-correia — APPROVED** [review]

File: helix-query.yaml [hunk 2]
```diff
@@ -68,12 +68,21 @@ indices:
...
+      endDate:
+        select: head > meta[property="end-date"]
+        value: attribute(el, "content")
       type:
         select: head > meta[name="event-type"]
         value: attribute(el, "content")
       region:
         select: head > meta[name="event-region"]
         value: attribute(el, "content")
+      city:
```
Comment (mhaack) [inline helix-query.yaml:83]: Should we better name this `address`? In `event-summary.js` we work with `address` as well, shouldn't have two names for the same thing.

Comment (lydiapuric) [reply helix-query.yaml:83]: agree, adjusting

**Review by mhaack — APPROVED** [review]

**Pull request merged**